Makefile

```diff
@@ -87,17 +87,17 @@ docs/%.5: docs/%.5.md
 
 docs: $(MANPAGES_MD:%.md=%)
 
-install: ocid.conf
+install:
```

Comment:
I don't personally do it that way, but do you really intend to do `make binaries docs ocid.conf && sudo make install` each time? The point is that doing it like this ensures that you don't install out-of-date binaries (though we need to fix up the dependencies of the binaries to include their sources).